src/error.js

```diff
@@ -63,6 +63,12 @@ const NON_ACTIONABLE_ERROR_THROTTLE_THRESHOLD = 0.001;
  */
 const USER_ERROR_THROTTLE_THRESHOLD = 0.1;
 
+/**
+ * Chance to post to the new error reporting endpoint.
+ * @const {number}
+ */
+const NEW_ERROR_REPORT_URL_FREQ = 0.2;
```

Comment:
20%?

Reply:
Yes; 20% of the 1% that will be using Beta/Experimental (along with 100% of mjs files in Beta/Experimental, since the legacy endpoint ignores errors in mjs files). See https://github.com/ampproject/error-tracker/issues/112 for full migration plan to new error reporting endpoint

Reply:
I believe we are flexible on this though

Reply:
@rcebulko just to confirm do we want to guard it to be only experimental? since the request was to cherry pick this to stable as well, there will be a 20% of stable that will also post to the new endpoint (unless i guard it a canary check)